core/src/test/java/org/apache/iceberg/rest/responses/TestLoadTableResponseParser.java

```diff
@@ -56,11 +58,12 @@ public void missingFields() {
         .hasMessage("Cannot parse missing field: metadata");
   }
 
-  @Test
-  public void roundTripSerde() {
+  @ParameterizedTest
+  @ValueSource(ints = {2, 3})
```

Comment:
We could just use @FieldSource("org.apache.iceberg.TestHelpers#ALL_VERSIONS")

Reply:
Or if we need to do a range --
```java
  private static int[] supportedInitialDefault =
      IntStream.rangeClosed(DEFAULT_VALUES_MIN_FORMAT_VERSION, MAX_FORMAT_VERSION).toArray();
      `1``
      
      Something like that ^

Reply:
@RussellSpitzer The V1 table format is pretty different. `schema`, `partition-spec` etc are required for V1, where V2+ uses `schemas`, `partitions-specs` etc. If you like I can add it as a separate test.

`DEFAULT_VALUES_MIN_FORMAT_VERSION` is private, but `MAX_FORMAT_VERSION` is a great suggestion 👍

Reply:
Ah sorry Default Values min format version was just an example, for this test you could probably just use 2 if you think 1 isn't worth testing in this suite.

Reply:
No worries, just explaining why I can't use that one. There is also no other constant available, so I've reinstated just `2`. I've added V1 as well for completeness 👍